airesources/Julia/MyBot.jl

```diff
@@ -17,8 +17,12 @@ using Memento
 logger = get_logger("bot-logger")
 
 game = Game("Settler")
+
+# Here one can do all preliminary checks and warmups, using initial 60s timeout.
```

Comment:
Can you have example of how to get the game_map here for initial inspection? Cpp bot for example prints basic map info such as player and planet counts.

Reply:
Yes, no problem at all. Added example.